torch/_refs/__init__.py

```diff
@@ -3551,6 +3552,21 @@ def t(a: TensorLikeType):
     return torch.transpose(a, 0, 0 if a.ndim < 2 else 1)
 
 
+# CompositeImplicitAutograd - don't register decomp
+def T(a: TensorLikeType) -> TensorLikeType:
+    # n != 2 && n != 0 is deprecated in regular PyTorch.  Maybe T could even be
+    # defined as an alias for t:
```

Comment:
`torch.t()` just returns tensors with fewer than two dimensions, so the behavior is distinct from `torch.Tensor.T`